procedures/release-duty/faq/faq.rst

```diff
@@ -0,0 +1,29 @@
+FAQ
+===
+
+This piece of docs tracks frequently-asked-questions during our releaseduty cycles.
+
+Adding a new locale into Product-Details
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+`Product details`_ is a public JSON API which contains release
+information for Mozilla products. More information on this can be found
+in `here`_.
+
+Each time a new locale is being added so that we’re able to
+ship a newly-localized Firefox in a different language, RelEng gets involved.
+In order for that to happen, the l10n team adds the corresponding builds as a
+pre-requisite. Then, they likely ping releaseduty: PR like `this`_ or
+`this <https://github.com/mozilla-releng/product-details/pull/9>`__ are
+opened against Product-details. TODO for releaseduty:
+
+
+- double-check that the PR is listing another locale in the ``public/1.0/languages.json`` file. See samples PRs like `this`_ or `this <https://github.com/mozilla-releng/product-details/pull/9>`__ from the past
+- review and merge the PR, either directly into ``production`` or via ``main`` and then follow-up with a push to ``production``
+- wait for the Taskcluster CI to run successfully. Behind the scenes, the CI is cloning the repo resources and pushes them to a S3 bucket
+- once the CI is green, trigger the Product-Details rebuild via Ship-it: |product-details-rebuild|
```

Comment:
Note to self: I may be wrong here - pending a conclusion in the PR conversation.

Reply:
Rebuilding in Ship It will override whatever we did manually to product-details (see above). We should think of this repo like a datastore, and nothing else.